### PR TITLE
chore(docs): extend SUPPORT.md to include guidelines on how to ask a support question about Vector

### PR DESCRIPTION
**Pull request by neuronull**
Porting what started as an entry in the GH wiki.

To be used in GH Discussions, and in Discord support.

### CODE REVIEW

**Review by spencergilbert — COMMENTED** [review]
I wonder if this should be written in a more neutral tone?

File: .github/SUPPORT.md [hunk 1]
```diff
@@ -1,5 +1,126 @@
 # Support
 
-Vector is a growing open source project with a variety of people willing to help you through various mediums.
+Vector is a growing open source project with a variety of people willing to help
+you through various mediums.
 
-Take a look at those mediums listed at https://vector.dev/community
+Take a look at those mediums listed at <https://vector.dev/community>
```
[the remark this answers is not in the document]
Comment (neuronull) [reply .github/SUPPORT.md:6]: It renders fine in the rich diff on the PR ...

It is a markdown lint.

https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md

[the remark this answers is not in the document]
Comment (spencergilbert) [reply .github/SUPPORT.md:6]: _Duh_

**Comment by neuronull** [comment]
> I wonder if this should be written in a more neutral tone?

It's definitely possible that I had a bit of a jaded tone come across when writing it.

Are there specific areas you think might benefit from improved wording?

File: .github/SUPPORT.md [hunk 2]
```diff
@@ -1,5 +1,126 @@
...
+<https://vector.dev/docs/>
+
+### Context and framing
+
+#### Provide details
+
+Essential details to include:
+
+- What is the version of Vector (and the Helm chart if deploying via Helm) and
+  the versions of any other systems in use (like Elasticsearch, NATs, etc.).
```
Comment (zamazan4ik) [inline .github/SUPPORT.md:37]: NATs -> NATS, like it is written here:  https://nats.io/

**Comment by spencergilbert** [comment]
> Are there specific areas you think might benefit from improved wording?

Once I land a PR or two on the log namespacing side I'll take a close look and provide some specific suggestions and/or places.

**Comment by jszwedko** [comment]
Great start! I took a pass over it to propose some changes: https://github.com/vectordotdev/vector/pull/16531. Given the scope of the changes, I figured a separate PR was more useful than trying to do it inline. Let me know what you think!

**Comment by spencergilbert** [comment]
👀 @jszwedko's proposed changes address (I think) any complaints I had

**Review by spencergilbert — APPROVED** [review]

**Review by zamazan4ik — APPROVED** [review]

**Review by jszwedko — APPROVED** [review]
🎉

**Comment by neuronull** [comment]
Thanks all!

**Pull request merged**